.github/workflows/tox.yml

```diff
@@ -181,4 +181,5 @@ jobs:
         if: ${{ contains(matrix.coverage, 'codecov') && matrix.pytest == 'true' }}
         uses: codecov/codecov-action@v3
         with:
-          token: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
Removing this should not make any difference because whether the default `secrets.CODECOV_TOKEN` is null or an empty string, it is interpreted the same as not being set at all: https://github.com/codecov/codecov-action/blob/e0fbd592d323cb2991fb586fdd260734fcb41fcb/src/buildExec.ts#L78-L80